examples/shapes/shapes_recursive_tree.c

```diff
@@ -0,0 +1,138 @@
+// Types and Structures Definition
+//----------------------------------------------------------------------------------
+typedef struct {
+    Vector2 start;
+    Vector2 end;
+    float angle;
+    float length;
+} Branch;
+
+//----------------------------------------------------------------------------------
+// Module Functions Declaration
+//----------------------------------------------------------------------------------
+static Vector2 CalculateBranchEnd(Vector2 start, float angle, float length);
+
+//------------------------------------------------------------------------------------
+// Program main entry point
+//------------------------------------------------------------------------------------
+int main(void)
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [shapes] example - shapes recursive tree");
+
+    Vector2 start = { (screenWidth / 2.0f) - 125, screenHeight };
+    float angle = 40.0f;
+    float thick = 1.0f;   
+    float treeDepth = 10.0f;
+    float branchDecay = 0.66f;
+    float length = 120.0f;
+    bool bezier = false;
+
+    SetTargetFPS(60);               // Set our game to run at 60 frames-per-second
+    //--------------------------------------------------------------------------------------
+
+    // Main game loop
+    while (!WindowShouldClose())    // Detect window close button or ESC key
+    {
+        // Update
+        //----------------------------------------------------------------------------------
+        float theta = angle * DEG2RAD;
+        int maxBranches = (int)(powf(2, (int)(treeDepth))); 
+        Branch branches[4096];
+        int count = 0;
+
+        Vector2 initialEnd = CalculateBranchEnd(start, 0.0f, length);
+        branches[count++] = (Branch){start, initialEnd, 0.0f, length};
+
+        for (int i = 0; i < count; i++) {
+            Branch branch = branches[i];
+            if (branch.length < 2) continue;
+
+            float nextLength = branch.length * branchDecay;
+
+            if (count < maxBranches && nextLength >= 2) {
+                Vector2 branchStart = branch.end;
+
+                Vector2 branchEnd1 = CalculateBranchEnd(branchStart, branch.angle + theta, nextLength);
+                Vector2 branchEnd2 = CalculateBranchEnd(branchStart, branch.angle - theta, nextLength);
+
+                branches[count++] = (Branch){branchStart, branchEnd1, branch.angle + theta, nextLength};
+                branches[count++] = (Branch){branchStart, branchEnd2, branch.angle - theta, nextLength};
+            }
+        }
+
+        // Draw
+        //----------------------------------------------------------------------------------
+        BeginDrawing();
+
+            ClearBackground(RAYWHITE);
+
+            for (int i = 0; i < count; i++) {
```

Comment:
Please, could you review it to follow raylib code conventions with aligned brackets?